versions.tf

```diff
@@ -1,9 +1,10 @@
 terraform {
-  required_version = ">=1.0.11, <=1.1.8"
+  required_version = ">=1.0.11, <=1.2.6"
+  
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.90.1"
+      version = "<=4.31.0"
```

Comment:
Did you intend to specify ">="
If not, you need to specify that the version should be above the previous one as well.

Reply:
Modified

Reply:
Modified